docs/docs/cookbook/cel.md

```diff
@@ -75,3 +78,27 @@ Cloud SDK installer and AI tools like claude code use osascript.
 
 Also if you're using osascript to do this legitimately this will break your
 usage.
+
+## Prevent users from enabling SSH (Santa 2025.8+)
+
+As called out in [loobins](https://www.loobins.io/binaries/systemsetup/) the
+systemsetup command can be used to enable SSH.
+
+To block this create a signing ID rule for `platform:com.apple.systemsetup` and
+attach the following CEL program:
+
+```clike
+args.join(" ").contains("-setremotelogin on")
```

Comment:
This would _only allow_ executions of systemsetup that are enabling SSH - a bool return of true is treated as ALLOWLIST. Better to be explicit like the other rules we have

```suggestion
args.join(" ").contains("-setremotelogin on") ? BLOCKLIST : ALLOWLIST
```